src/app/core/learning-object-module/learning-object/learning-object.routes.ts

```diff
@@ -38,6 +38,23 @@ export const LEARNING_OBJECT_ROUTES = {
     },
 };
 
+export const USER_ROUTES = {
+    GET_MY_LEARNING_OBJECTS(
+      username,
+      filters: any,
+      query: string,
+    ) {
+        // Onion
+        let uri = `${environment.apiURL}/users/${encodeURIComponent(
+            username,
+        )}/learning-objects?${querystring.stringify(filters)}`;
+        if (query) {
+          uri += `&text=${encodeURIComponent(query)}`;
+        }
```

Comment:
Is the query just the text search? We can add that to the filters type and then just stringify that instead of having an if in here.